install_config/install/deploy_router.adoc

```diff
@@ -1029,23 +1029,23 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ----
 ====
 
-
+[[preventing-connection-failures-during-restarts]]
 === Preventing Connection Failures During Restarts
 
 If you connect to the router while the proxy is reloading, there is a
 small chance that your connection will end up in the wrong network
-queue and be dropped.  The issue is being addressed.  In the meantime,
-it is possible to work around the problem by installing *_iptables_*
-rules to prevent connections during the reload window.  However, doing
+queue and be dropped. The issue is being addressed. In the meantime,
+it is possible to work around the problem by installing `iptables`
+rules to prevent connections during the reload window. However, doing
 so means that the router needs to run with elevated privilege so that
-it can manipulate *_iptables_* on the host.  It also means that
+it can manipulate `iptables` on the host. It also means that
 connections that happen during the reload are temporarily ignored and
 will have to retransmit their connection start, lengthening the time
 it takes to connect, but preventing connection failure.
 
 Due to these issues, do not enable this option most of the time.
 However, if you decide you must try to prevent this case from
```

Comment:
overly verbose:

"if you decide you must try to prevent this case from happening"

How about just "To prevent this, you can make..."